EIPS/eip-1679.md

```diff
@@ -40,6 +40,7 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
 - [EIP-1702](https://eips.ethereum.org/EIPS/eip-1702): Generalized account versioning scheme
 - [EIP-1803](https://eips.ethereum.org/EIPS/eip-1803): Rename opcodes for clarity
 - [EIP-1884](https://eips.ethereum.org/EIPS/eip-1884): Repricing for trie-size-dependent opcodes
+- [EIP-1930](https://github.com/ethereum/EIPs/pull/1950): Strict Gas CALL
```

Comment:
Since #1950 was merged, can you please update this to https://eips.ethereum.org/EIPS/eip-1950?